code/node-server/src/middleware/validator.js

```diff
@@ -18,6 +18,7 @@ const routeValidators = {
     '/board': [validID],
     '/groups': [validID],
     '/tile': [validTile, validID],
+    '/netstat': [() => true],
```

Comment:
Not a great validation for the `/netstat` endpoint but needed it in order to make GET requests here